### PR TITLE
Basic methods - Events

### PR DESCRIPTION
**Pull request by ristovska1606**
Added basic methods. Will be modified if it is necessary.

### CODE REVIEW

File: src/backend/TeachersSideAPI/TeachersSideAPI/Service/Implementation/EventService.cs [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+    public async Task<IEnumerable<EventDto>> GetAll()
+    {
+        IEnumerable<Event> events = await _eventRepository.GetAll();
+        return _mapper.Map<IEnumerable<EventDto>>(events);
+    }
+
+    public async Task<EventDto?> Get(int id)
+    {
+        var evt = await _eventRepository.Get(id)
+            ?? throw new EventNotFoundException($"Event with ID:{id} not found.");
```
Comment (EkaterinaSazdova) [inline src/backend/TeachersSideAPI/TeachersSideAPI/Service/Implementation/EventService.cs:31]: Why throw an exception here? Return type is EventDto? and the repository method returns first or default so it's not needed. Avoid throwing exceptions in async methods if not needed.

File: src/backend/TeachersSideAPI/TeachersSideAPI/Web/Controllers/EventsController.cs [hunk 2]
```diff
@@ -10,9 +14,52 @@ namespace TeachersSideAPI.Web.Controllers;
...
+    {
+        try
+        {
+            var evt = await _eventService.Get(id);
+            return Ok(evt);
+        }
+        catch (EventNotFoundException exception)
+        {
+            return NotFound(exception.Message);
+        }
```
Comment (EkaterinaSazdova) [inline src/backend/TeachersSideAPI/TeachersSideAPI/Web/Controllers/EventsController.cs:42]: Same here, remove the try-catch block and return NotFound if resulting dto is null.

Comment (EkaterinaSazdova) [reply src/backend/TeachersSideAPI/TeachersSideAPI/Web/Controllers/EventsController.cs:42]: For example something like this`
var event = await _eventService.Get(id);
return (event == null) ?  NotFound() :  Ok(event);
`

**Comment by SazdovaEkaterina** [comment]
[Development for Events Page](https://trello.com/c/UWFt006Z/13-development-for-events-page)

**Review by SazdovaEkaterina — APPROVED** [review]

**Pull request merged**